hack/make-release-notes.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+set -o errexit
+set -o pipefail
+
+TOOLS_FOLDER=${1:-../hack/tools/bin}
+
+mkdir $TOOLS_FOLDER/cluster-api
+cd $TOOLS_FOLDER/cluster-api
```

Comment:
I think this script has to be adjusted for rancher turles

Reply:
I'm not sure I understand. This script was created for rancher turtles specifically, what I missing here to be adjusted?

Reply:
Ok, I see there is no way to import the release note tool here